### PR TITLE
#68 & #116 - Applying fullwidth to mobile navs, adding schedule mobile nav

### PR DESCRIPTION
**Pull request by sergesemashko**
@likewhoa please take a look

### CODE REVIEW

**Comment by likewhoa** [comment]
I'll be going to playday @Sergey-Semashko , lets go over this when i get there. Hope to see you there!

**Comment by sergesemashko** [comment]
@likewhoa unfortunately, I can't do this today. But I'm available in IRC or Skype till 4pm, please take a look at the behavior http://monosnap.com/file/YjUDgjgs5XM6QOzKkqhMbhUCuhrjP9 .

File: sites/all/themes/zen_avenue/css/avenue.css [hunk 1]
```diff
@@ -425,7 +425,6 @@ h2.title-inspiration, .node-type-blog h1#page-title {
 	-webkit-box-shadow: inset -2px 0 2px #06253e;
 	box-shadow: inset -2px 0 2px #06253e;
 	background-color: #06253e;
-	bottom: 0;
 	color: #eee;
 	font-size: 13px;
 	margin-bottom: -99999px;
```
Comment (likewhoa) [inline sites/all/themes/zen_avenue/css/avenue.css:430]: do we really need this still?

**Comment by likewhoa** [comment]
Since you put the slide out menu to be under the top menus, there really is not point in having a title 'NYC Camp' repeated, so remove this since it's already on top.

One other suggestion but not a blocker would be to add swipe support for mobile, so in addition to clicking on the menu icon we can have menus be swiped out by left and right gesture swipes.

File: sites/all/themes/zen_avenue/css/avenue.css (hunk 1 above)
Comment (sergesemashko) [inline sites/all/themes/zen_avenue/css/avenue.css:430]: Actually, yes, without this trick schedule bar is not full height http://monosnap.com/image/255rcToEEzKFauXBr0DrPPm28eNLQe

**Comment by sergesemashko** [comment]
@likewhoa please review

**Comment by esod** [comment]
@Sergey-Semashko I don't know too much about this. Will you confirm that it is okay on dev? Thanks

**Comment by sergesemashko** [comment]
@esod could you please share the link to env you mentioning or I should create it by myself?

**Comment by esod** [comment]
@Sergey-Semashko You can see it on live.

**Pull request merged**